api/services/ocean/spark/paths/workspaceStart.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Ocean Spark
+
+put:
+  summary: "Start Workspace"
+  description: >
+    All Ocean Spark configuration parameters are updatable. This API supports partial updates, so specific fields can be updated separately.
+  operationId: "OceanSparkWorkspaceStart"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../parameters/path/oceanSparkWorkspaceId.yaml"
+    - $ref: "../../../../commons/parameters/accountId.yaml"
+  requestBody:
```

Comment:
I don't remember we need to submit a body when starting a workspace, workspaceId is passed as path param and accountId as queryParam